srcCpp/CustomType.cxx

```diff
@@ -0,0 +1,102 @@
+/*
+ * (c) 2005-2018  Copyright, Real-Time Innovations, Inc. All rights reserved.
+ * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
+ */
+#include "CustomType.h"
+/*
+ * It is the source code files that contain the implementation of the requires
+ * and optional functions used for the handling of the custom type. These
+ * functions will be used only if the parameter "--customType <type>" is aplpied
+ * in the build. You can find more information in the point "Using Custom Types"
+ * of the section "Use-Cases And Examples" in the documentation.
+ */
+
+/*
+*   This function is used to initialize your data.
+*   @param data: A reference to the customer type.
+*   @return true if the operation was success, otherwise false.
+*/
+bool initialize_custom_type(RTI_CUSTOM_TYPE &data)
+{
+    return true;
+}
```

Comment:
@brief 
for @param specify if input or output
@param data: A reference to the customer type object that will be sent on the wire.